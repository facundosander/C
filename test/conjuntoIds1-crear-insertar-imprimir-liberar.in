# crea un conjunto con rango 1..10
crearConjunto 10

# imprime el conjunto vacio, esto es, una linea vacia
imprimirConjunto

insertarConjunto 3
# imprime un conjunto con 3 como unico elemento
imprimirConjunto

insertarConjunto 5
insertarConjunto 7
# imprime conjunto con 3 elementos
imprimirConjunto

insertarConjunto 1
insertarConjunto 10
# imprimir conjunto con casos borde
imprimirConjunto

# intenta insertar elementos fuera de rango
insertarConjunto 0
insertarConjunto 15
# imprime el conjunto sin modificaciones
imprimirConjunto

# libera el conjunto
liberarConjunto
Fin