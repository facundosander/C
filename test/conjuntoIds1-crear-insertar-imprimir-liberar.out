1>#  crea un conjunto con rango 1..10.
2>3>#  imprime el conjunto vacio, esto es, una linea vacia.
4>
5>6>#  imprime un conjunto con 3 como unico elemento.
7>3 
8>9>10>#  imprime conjunto con 3 elementos.
11>3 5 7 
12>13>14>#  imprimir conjunto con casos borde.
15>1 3 5 7 10 
16>#  intenta insertar elementos fuera de rango.
17>18>19>#  imprime el conjunto sin modificaciones.
20>1 3 5 7 10 
21>#  libera el conjunto.
22>23>Fin.
